### PR TITLE
fix(docker): revert CMD

### PR DESCRIPTION
**Pull request by WikiRik**

### CODE REVIEW

**Review by serge1peshcoff — APPROVED** [review]

**Comment by WikiRik** [comment]
You can ignore the linter for now, just need to ensure that statutory is running again

File: docker/statutory/Dockerfile [hunk 1]
```diff
@@ -23,6 +23,6 @@ RUN npm install -g bunyan@1.8.15
 RUN npm cache clean --force
 RUN npm install
 
-CMD ["sh", "-c", "/usr/app/scripts/bootstrap.sh && nodemon -e 'js,json' lib/run.js"]
```
Comment (WikiRik) [inline docker/statutory/Dockerfile:26]: Okay, running it like this works on my local server (not my laptop, but a Ubuntu server with no vagrant). Even after removing the files and the container and rebuilding it from the start. Any ideas on how to test this properly? @serge1peshcoff

Comment (serge1peshcoff) [reply docker/statutory/Dockerfile:26]: no idea unfortunately, we can deploy once more and check if it works

Comment (WikiRik) [reply docker/statutory/Dockerfile:26]: I would be surprised if that works, but we can try. Can you deploy tag 1.6.8?

Comment (serge1peshcoff) [reply docker/statutory/Dockerfile:26]: just checked, it doesn't fix it somehow

Comment (WikiRik) [reply docker/statutory/Dockerfile:26]: But 1.6.9 does work? I reverted the CMD there

Comment (serge1peshcoff) [reply docker/statutory/Dockerfile:26]: yes, just checked, this works

Comment (WikiRik) [reply docker/statutory/Dockerfile:26]: Okay, so this change breaks stuff. @linuxbandit do you have any clue why? The code on the left gives a 'permission denied' error, but the code on the right works. Shouldn't they be executed by the same user?

**Pull request merged**